runtime/test-runtime/src/xcm_config.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use frame_support::{parameter_types, traits::All, weights::Weight};
+use xcm::latest::{
+	Error as XcmError, Junction::*, MultiAsset, MultiLocation, MultiLocation::*, NetworkId,
+	Result as XcmResult, SendXcm, Xcm,
+};
+use xcm_builder::{AllowUnpaidExecutionFrom, FixedWeightBounds, SignedToAccountId32};
+use xcm_executor::{
+	traits::{InvertLocation, TransactAsset, WeightTrader},
+	Assets,
+};
+
+parameter_types! {
+	pub const OurNetwork: NetworkId = NetworkId::Polkadot;
+}
+
+/// Type to convert an `Origin` type value into a `MultiLocation` value which represents an interior location
+/// of this chain.
+pub type LocalOriginToLocation = (
+	// And a usual Signed origin to be used in XCM as a corresponding AccountId32
+	SignedToAccountId32<crate::Origin, crate::AccountId, OurNetwork>,
+);
+
+pub struct DoNothingRouter;
+impl SendXcm for DoNothingRouter {
+	fn send_xcm(_dest: MultiLocation, _msg: Xcm<()>) -> XcmResult {
+		Ok(())
+	}
+}
+
+pub type Barrier = AllowUnpaidExecutionFrom<All<MultiLocation>>;
```

Comment:
This is better than `struct YesItShould` that I put for benchmarking mock, perhaps use this there as well: https://github.com/paritytech/polkadot/pull/3563/files#diff-4956fec0e6b5a0cf3ad1a1980b05285ac79c5a91d5edb15863d19b1519e8ae40R21